saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -1,6 +1,19 @@
 #!/bin/bash
 
 set -ex
+
+ACC_KEY_TMP_FILE=$(mktemp)
+chmod 600 "$ACC_KEY_TMP_FILE"
+trap 'rm -f "$ACC_KEY_TMP_FILE"' EXIT
+
+cat > ${ACC_KEY_TMP_FILE} <<EOF
+CCM_V2_AGENT_ACCESS_KEY_ID=$CCM_V2_AGENT_ACCESS_KEY_ID
+CCM_V2_AGENT_ENCIPHERED_ACCESS_KEY=$CCM_V2_AGENT_ENCIPHERED_ACCESS_KEY
+EOF
+
+unset CCM_V2_AGENT_ACCESS_KEY_ID
+unset CCM_V2_AGENT_ENCIPHERED_ACCESS_KEY
```

Comment:
I suggest not doing this. None of the other environment variables are unset. Their values are only visible inside `user-data-helper.sh` (and its child processes), and that is okay.